src/test/java/co/cask/hydrator/salesforce/plugin/source/batch/SalesforceSourceConfigBuilder.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator.salesforce.plugin.source.batch;
+
+/**
+ * Helper class to simplify {@link SalesforceSourceConfig} class creation.
+ */
+public class SalesforceSourceConfigBuilder {
```

Comment:
For basic boilerplate classes like this, it may be helpful to use autovalue in the future.

https://github.com/google/auto/tree/master/value